lakefile.lean

```diff
@@ -77,3 +80,49 @@ script setup do
   | .err res => IO.eprintln res; return 1
 
 end Setup
+
+section ImportAll
+
+open System
+
+partial def getLeanFilePathsList (fp : FilePath) (acc : Array FilePath := #[]) :
+    IO $ Array FilePath := do
+  if ← fp.isDir then
+    let mut extra : Array FilePath := #[]
+    for dirEntry in ← fp.readDir do
+      for innerFp in ← getLeanFilePathsList dirEntry.path do
+        extra := extra.push innerFp
+    return acc.append extra
+  else
+    if (fp.extension.getD "") = "lean" then
+      return acc.push fp
+    else
+      return acc
+
+open Std (RBTree)
+
+def getAllFiles : ScriptM $ List String := do
+  let paths := (← getLeanFilePathsList ⟨"Yatima"⟩).map toString
+  let paths : RBTree String compare := RBTree.ofList (paths.data) -- ordering
```

Comment:
I think you can just write `List.sort paths` here; it's more clear

Reply:
We don't have `List.sort` available on the lakefile :(